EasyPost/Models/API/Batch.cs

```diff
@@ -115,6 +115,7 @@ public async Task<Batch> GenerateScanForm(string fileFormat = "pdf")
         [CrudOperations.Update]
         public async Task<Batch> RemoveShipments(Dictionary<string, object> parameters)
         {
+            // parameters = parameters.Wrap("batch");  // TODO: Update docs to remove wrapped "batch" key
```

Comment:
`batch` should be wrapped and included here

Reply:
Same as https://github.com/EasyPost/easypost-csharp/pull/360#discussion_r1003569828